src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index];
```

Comment:
The return type of 'getRandomColor()' method should be Color instead of String because you are returning an instance of the Color enum. Change the return type to Color to match the type of the object you are returning.